safenode/src/client/wallet.rs

```diff
@@ -34,9 +34,15 @@ impl<W: SendWallet> WalletClient<W> {
     }
 
     /// Send tokens to another wallet.
-    pub async fn send(&mut self, amount: Token, to: PublicAddress) -> Result<Vec<CreatedDbc>> {
+    pub async fn send(&mut self, amount: Token, to: PublicAddress) -> Result<Dbc> {
```

Comment:
What is the reason behind changing this interface?

Reply:
The `CreatedDbc` struct contains `RevealedAmount`, which is useful to know what the amount in the dbc actually was (as it's blinded once created, and only the recipient will be able to see it).
But this `RevealedAmount` type is a bit low level to be part of that interface, and isn't very useful at that level. It should be part of the wallet ledger I think, where it will be automatically accounted for in a user friendly manner (and in later versions, they could export it from wallet api if wishing so).

Reply:
What is the reason behind changing this interface?